tests/main/document-portal-activation/task.yaml

```diff
@@ -17,75 +17,75 @@ description: |
 # Disabled on Ubuntu Core because it doesn't provide the "desktop"
 # slot, and Amazon Linux because it doesn't have the required Python 3
 # packages to run the test.
-systems: [ -ubuntu-core-*, -amazon-linux-2-*, -centos-* ]
-
-environment:
-    XDG_RUNTIME_DIR: /run/user/$(id -u)
```

Comment:
Would it make sense to have a variable for /run/user/12345/ instead?

Reply:
I personally don't think so, it tends to be quirky or annoying with quoting, necessitating the the use of shellcheck disable comments. It also is heavier cognitively, requiring the reader to jump back and forth.

In addition the variable should never have been called `XDG_RUNTIME_DIR`, as that interferes with legitimate use of the variable already present in the `bash` process executing the test and can lead to confusing and hard-to-debug failures.